tagstudio/src/core/library/alchemy/library.py

```diff
@@ -438,6 +439,18 @@ def search_library(
                 )
             elif search.path:
                 statement = statement.where(Entry.path.ilike(f"%{search.path}%"))
+            elif search.filetype:
+                statement = statement.where(Entry.suffix.ilike(f"%{search.filetype}%"))
```

Comment:
This introduces a quirk where filetypes containing the name of others are returned along with the original search. As an example, I had `.apng` files returned alongside `.png` files when searching for `filetype:png`

Reply:
fixed that and added a test that should catch that if it ever happens in the future